101-vm-trustedlaunch-linux/azuredeploy.json

```diff
@@ -76,7 +76,7 @@
     },
     "location": {
       "type": "string",
-      "defaultValue": "[resourceGroup().location]",
+      "defaultValue": "southcentralus",
```

Comment:
you cannot default to a fixed location... if the sample does not work in all locations state that in the readme file and use allowedValues.   If you need a fixed location for the test you can use the param file.

Reply:
Done

Reply:
You need set allowedValues to the supported locations